react-frontend/src/sharedComponent/Hi5CountSharedComponent.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import LabelCoreComponent from "coreComponents/LabelCoreComponent";
+import ImageCoreComponent from "coreComponents/ImageCoreComponent";
+const Hi5CountSharedComponent = ({
```

Comment:
rename this to `Hi5CountSharedComponent` to `Hi5CountComponent` its already under shared directory no need to specify